lib/docs/scrapers/react_bootstrap.rb

```diff
@@ -0,0 +1,34 @@
+module Docs
+  class ReactBootstrap < UrlScraper
+    self.slug = 'react_bootstrap'
+    self.type = 'simple'
+    self.release = '1.5.0'
+    self.base_url = 'https://react-bootstrap.github.io/'
+
+    self.links = {
+      home: 'https://react-bootstrap.github.io',
+      code: 'https://github.com/react-bootstrap/react-bootstrap'
+    }
+
+    html_filters.push 'react_bootstrap/entries', 'react_bootstrap/clean_html'
+
+    options[:skip] = %w(
+      react-overlays/
+    )
+
+    options[:replace_paths] = {
+    }
+
+    options[:trailing_slash] = true
+
+    options[:attribution] = <<-HTML
```

Comment:
Please also add the license to `assets/javascripts/templates/pages/about_tmpl.coffee`